src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
     }
 
-    function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+    function closeForm1() {
+      document.getElementById("myForm1").style.display = "none";
+    }
+
+    function closeForm2() {
+        document.getElementById("myForm2").style.display = "none";
```

Comment:
Created separate JS functions for different forms. Could likely be combined into one function if parameter is passed in.